schema/INDIVIDUAL_RECEIPT_NON_CONTRIBUTION_ACCOUNT.json

```diff
@@ -0,0 +1,581 @@
+            "minLength": 0,
+            "maxLength": 9,
+            "pattern": "^[ -~]{0,9}$",
+            "examples": [
+                981110123
+            ],
+            "fec_spec": {
+                "FIELD_DESCRIPTION": "CONTRIBUTOR ZIP",
+                "TYPE": "A/N-9",
+                "REQUIRED": "X (error)",
+                "SAMPLE_DATA": 981110123,
+                "VALUE_REFERENCE": null,
+                "RULE_REFERENCE": null,
+                "FIELD_FORM_ASSOCIATION": null
+            }
+        },
+        "contribution_date": {
+            "title": "CONTRIBUTION DATE",
+            "description": "",
+            "type": "string",
+            "minLength": 10,
+            "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}$",
+            "examples": [
+                "2018-11-13"
+            ],
+            "fec_spec": {
+                "FIELD_DESCRIPTION": "CONTRIBUTION DATE",
+                "TYPE": "NUM-8",
+                "REQUIRED": "X (error)",
+                "SAMPLE_DATA": 20120615,
+                "VALUE_REFERENCE": null,
+                "RULE_REFERENCE": null,
+                "FIELD_FORM_ASSOCIATION": null
+            }
+        },
+        "contribution_amount": {
+            "title": "CONTRIBUTION AMOUNT",
+            "description": "",
+            "type": "number",
+            "minimum": -99999999.99,
+            "maximum": 999999999.99,
+            "examples": [
+                250
+            ],
+            "fec_spec": {
+                "FIELD_DESCRIPTION": "CONTRIBUTION AMOUNT",
+                "TYPE": "AMT-12",
+                "REQUIRED": "X (error)",
+                "SAMPLE_DATA": 250,
+                "VALUE_REFERENCE": null,
+                "RULE_REFERENCE": null,
+                "FIELD_FORM_ASSOCIATION": null
+            }
+        },
+        "contribution_aggregate": {
+            "title": "CONTRIBUTION AGGREGATE",
+            "description": "",
+            "type": "number",
+            "minimum": -99999999.99,
+            "maximum": 999999999.99,
+            "examples": [
+                1000
+            ],
+            "fec_spec": {
+                "FIELD_DESCRIPTION": "CONTRIBUTION AGGREGATE",
+                "TYPE": "AMT-12",
+                "REQUIRED": "X (error)",
+                "SAMPLE_DATA": 1000,
+                "VALUE_REFERENCE": null,
+                "RULE_REFERENCE": "YTD",
+                "FIELD_FORM_ASSOCIATION": null
+            }
+        },
+        "aggregation_group": {
+            "title": "AGGREGATION GROUP",
+            "description": "",
+            "const": "NON_CONTRIBUTION_ACCOUNT",
+            "examples": [
+                "NON_CONTRIBUTION_ACCOUNT"
+            ],
+            "fec_spec": {
+                "FIELD_DESCRIPTION": "AGGREGATION GROUP",
+                "TYPE": "A/N-100",
+                "REQUIRED": "X (error)",
+                "SAMPLE_DATA": "NON_CONTRIBUTION_ACCOUNT",
+                "VALUE_REFERENCE": null,
```

Comment:
Value/rule reference should be "Non-Contribution Account"

Reply:
Addressed